script/generate_imports.py

```diff
@@ -1,18 +1,17 @@
 import sys
 from os import listdir, getcwd
-from os.path import abspath, isdir, join
+from os.path import abspath, isdir, join, dirname, basename
 from argparse import ArgumentParser
 
 sys.path.append(abspath("scripts"))
 from generate_imports_helper import comment, get_importable_lines
 
 
 import_line_format = "\t_ \"{beat_path}/{module}/{name}\""
-include_list_path = "include/list.go"
 import_template = """/*
 {comment}
 */
-package include
+package {package}
```

Comment:
I wonder if this is going to break other beats.

BTW: you need to run `make fmt` to make CI happy, then will see ;-)

Reply:
package name is computed from file path. It shouldn't break the other beats.